File: PAG/app/Clases/OBJETOS_DEL_GASTO_DTO.ts



    // $Classes/Enums/Interfaces(filter)[template][separator]
    // filter (optional): Matches the name or full name of the current item. * = match any, wrap in [] to match attributes or prefix with : to match interfaces or base classes.
    // template: The template to repeat for each matched item
    // separator (optional): A separator template that is placed between all templates e.g. $Properties[public $name: $Type][, ]

    // More info: http://frhagn.github.io/Typewriter/

    
    export class OBJETOS_DEL_GASTO_DTO {
        
        // GESTION
        public GESTION: number = 0;
        // OBJETO
        public OBJETO: string = null;
        // GRUPO_OBJETO
        public GRUPO_OBJETO: number = 0;
        // SUB_GRUPO_OBJETO
        public SUB_GRUPO_OBJETO: number = 0;
        // PARTIDA_OBJETO
        public PARTIDA_OBJETO: number = 0;
        // SUB_PARTIDA_OBJETO
        public SUB_PARTIDA_OBJETO: number = 0;
        // DESC_OBJETO
        public DESC_OBJETO: string = null;
        // VIGENTE
        public VIGENTE: string = null;
        // API_ESTADO
        public API_ESTADO: string = null;
    }

File: PAG_DTO/OBJETOS_DEL_GASTO_DTO.cs
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;

namespace PAG_DTO
{
    public class OBJETOS_DEL_GASTO_DTO
    {
        [DisplayName("Gestión")]//, MaxLength(4), ReadOnly(true)]
        public short GESTION { get; set; }
        [DisplayName("Objeto")]//, MaxLength(5), ReadOnly(true)]
        public string OBJETO { get; set; }
        [DisplayName("Grupo Objeto")]//, MaxLength(30), ReadOnly(true)]
        public short GRUPO_OBJETO { get; set; }
        [DisplayName("Sub Grupo Objeto")]//, MaxLength(1), ReadOnly(true)]
        public short SUB_GRUPO_OBJETO { get; set; }
        [DisplayName("Partida")]//, MaxLength(1), ReadOnly(true)]
        public short PARTIDA_OBJETO { get; set; }
        [DisplayName("Sub Partida")]//, MaxLength(2), ReadOnly(true)]
        public short SUB_PARTIDA_OBJETO { get; set; }
        [DisplayName("Institución")]//, MaxLength(200), ReadOnly(true)]
        public string DESC_OBJETO { get; set; }
        [DisplayName("Institución")]//, MaxLength(1), ReadOnly(true)]
        public string VIGENTE { get; set; }
        [DisplayName("Institución")]//, MaxLength(20), ReadOnly(true)]
        public string API_ESTADO { get; set; }
    }
}
